btk/utils.py

```diff
@@ -306,7 +306,7 @@ def make_true_seg_map(image, threshold):
     return seg_map.astype(np.bool)
 
 
-def basic_selection_function(catalog):
+def basic_selection_function(catalog, max_size = 4, max_mag = 27):
```

Comment:
way simpler, thanks. It will be easier to decide later if we want to keep selection functions